platform/podjs/src/browserPod.ts

```diff
@@ -317,6 +401,15 @@ interface NetworkResponse {
 }
 
 class BrowserNetwork {
+    /**
```

Comment:
We will probably need to adjust podjs now...

Reply:
@JJ I'm not sure with this diff on github where this comment refers to 💭

Reply:
Right, it's probably outdated.